cmd/support-notifications/main.go

```diff
@@ -85,7 +84,7 @@ func main() {
 		return
 	}
 
-	r := notifications.LoadRestRoutes()
+	//r := notifications.LoadRestRoutes()
```

Comment:
This line commented out intentionally. It's redundant with content in the startHttpServer() function. I should have removed the line before commit. Perhaps @akramtexas can remove it.

Reply:
Good point @tsconn23 (Done, earlier. At this time, adding a comment here for acknowledgment and closure).